opentracing_utils/tracers.py

```diff
@@ -51,6 +51,8 @@ def init_opentracing_tracer(tracer, **kwargs):
         collector_encryption = "tls"
         if scheme == "http":
             collector_encryption = "none"
+        if 'collector_encryption' in kwargs:
```

Comment:
hmm, I would expect that we give kwargs higher precedence.

Reply:
we do?

Reply:
Well, if the caller passes `collector_encryption` then I would assume they know what they are doing, and implicitly changing that value by the lib could be confusing for them.

Reply:
isn't that what `collector_encryption = kwargs.pop('collector_encryption', collector_encryption)` does?

Reply:
Ah right 🤦 !